### DISCUSSION TITLE
请问：启用滚动，移动节点到滚动条附件，如何自动滚动？

### DISCUSSION THREAD

**Discussion by Mrheqiang** [antvis/X6#3391] [Q&A]
v1中，在启用了滚动时，当移动节点到画布边缘的时候，如何自动将画布不断滚动。

**Comment by book12085** [comment]
同问

**Reply by NewByVector** [reply]
默认就会自动滚动呀，参考官网 [demo](http://x6.antv.antgroup.com/tutorial/plugins/scroller)。

**Reply by Mrheqiang** [reply]
我想得到的效果是：当节点在画布外的时候，滚动条就要不断的移动。（只要节点在外面，滚动条就要不停的滚动）现在效果是在外面也需要不断的动。

**Reply by NewByVector** [reply]
鼠标不移动咋知道画布移动到哪里，什么时候停止呢？

**Reply by Mrheqiang** [reply]
当把节点放在画布中  或者放开鼠标就停止
